docker/run_update_data.sh

```diff
@@ -8,14 +8,22 @@ set -eo pipefail
 
 # Fetch and update release information for these products (comma-delimited)
 PRODUCTS="firefox"
+CRONTABBER="docker-compose run crontabber python socorro/cron/crontabber_app.py"
+
 
 # Fetch release data
-docker-compose run crontabber python socorro/cron/crontabber_app.py --job=ftpscraper \
+${CRONTABBER} --job=ftpscraper \
        --crontabber.class-FTPScraperCronApp.products=${PRODUCTS}
 
 # Update featured versions data based on release data
-docker-compose run crontabber python socorro/cron/crontabber_app.py --job=featured-versions-automatic \
+${CRONTABBER} --job=featured-versions-automatic \
        --crontabber.class-FeaturedVersionsAutomaticCronApp.products=${PRODUCTS}
 
 # Fetch normalization data for versions we know about
 docker-compose run processor python docker/fetch_normalization_data.py
+
+# Truncate tables
+${CRONTABBER} --job=truncate-partitions
+
+# Create new weekly tables
+${CRONTABBER} --job=weekly-reports-partitions
```

Comment:
This one creates `raw_crashes_yyyymmdd` tables which we need for storing crashes. Otherwise the postgres crashstorage fails epically.